src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ObjectTypeVisitor.cs

```diff
@@ -213,7 +205,7 @@ private void ProcessProperties(IOpenApiSchemaAcceptor instance, string schemaNam
                                               .Select(p => p.First())
                                               .ToDictionary(p => p.Value.Title, p => p.Value);
 
-            foreach (var schema in schemasToBeAdded.Where(p => p.Key != "jObject" && p.Key != "jToken"))
+            foreach (var schema in schemasToBeAdded.Where(p => !this._noAddedKeys.Contains(p.Key)))
```

Comment:
Can we use `p.Key.ToLowerInvariant()` or `p.Key.ToUpperInvariant()` here?

Reply:
I used UPPERCASE for the key, thanks